Core/Core/API/APICourseRequestable.swift

```diff
@@ -24,13 +24,20 @@ public struct GetCoursesRequest: APIRequestable {
 
     public enum State: String {
         case available, completed, unpublished, deleted
+
+        public static var defaults: [State] {
+            if Bundle.main.isTeacherApp {
```

Comment:
Anyone have thoughts on how to override this for tests?

Reply:
Maybe `isTeacherApp` should also be true if the the main bundle is the runner?

Reply:
Good idea. Done.